### PR TITLE
Update checklist for post-inclusion recommendations

### PR DESCRIPTION
**Pull request by samccann**
Found this older issue so opening a PR so we can discuss...

Fixes https://github.com/ansible-community/community-team/issues/100

### CODE REVIEW

File: collection_checklist.md [hunk 1]
```diff
@@ -54,3 +54,10 @@ Note for reviewers: If you don't know how to check the points below, please ask
 - [ ] all CI tests MUST run against every pull request
 - [ ] all CI tests MUST run regularly (nightly, or at least once per week)
 - [ ] sanity tests MUST run against a commit that releases the collection; if they don't pass, the collection won't be released
+
+**Steps to complete after inclusion is approved**
+
+- [ ] Subscribe to the [news-for-maintainers](https://github.com/ansible-collections/news-for-maintainers) repo and track announcements there.
+- [ ] Optionally but highly recommended, subscribe to the [Ansible Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn). You can also post in the Bullhorn for community feedback or for important announcements such new releases of this collection.
+- [ ] Review the [Maintainer guidelines](https://github.com/ansible/community-docs/blob/main/maintaining.rst). 
+- [ ] Optionally, share the above with future maintainers, in a README on CONTRIBUTING file in the collection repo.
```
Comment (felixfontein) [inline collection_checklist.md:63]: I don't think this should be part of this checklist. This checklist is for reviewing only so far. Maybe we should add this as a separate file?

Comment (Andersson007) [reply collection_checklist.md:63]: - Adding as a separate file as @felixfontein suggested SGMT
- I wouldn't include the news-for-maintainers at least now
- All the other links should be updated

File: collection_checklist.md [hunk 2]
```diff
@@ -54,3 +54,10 @@ Note for reviewers: If you don't know how to check the points below, please ask
 - [ ] all CI tests MUST run against every pull request
 - [ ] all CI tests MUST run regularly (nightly, or at least once per week)
 - [ ] sanity tests MUST run against a commit that releases the collection; if they don't pass, the collection won't be released
+
+**Steps to complete after inclusion is approved**
```
Comment (Andersson007) [inline collection_checklist.md:58]: Added couple of points, thanks for pinging!

**Comment by Andersson007** [comment]
the doc has been moved to https://github.com/ansible-collections/ansible-inclusion/blob/main/collection_checklist.md, could you please re-create the PR there? Closing this one, thanks!

**Pull request closed without merging**